python/servo/package_commands.py

```diff
@@ -674,6 +677,22 @@ def upload_to_s3(platform, package, timestamp):
                 }
             )
 
+            if github_release_id:
```

Comment:
This seems fairly independent of `upload_to_s3` so perhaps create a separate `upload_to_github_release` function and use an early return if there is no if github_release_id.

Reply:
Makes sense. Addressed in 61fb795. I've also extracted the package hash computation to be outside of upload_to_s3.